src/app-layout/visual-refresh-toolbar/index.tsx

```diff
@@ -116,27 +111,15 @@ const AppLayoutVisualRefreshToolbar = React.forwardRef<AppLayoutProps.Ref, AppLa
       //   and compare a given number with the new drawer id min size
 
       // the total size of all global drawers resized to their min size
-      let totalActiveDrawersMinSize = activeGlobalDrawersIds
-        .map(
-          activeDrawerId => combinedDrawers.find(drawer => drawer.id === activeDrawerId)?.defaultSize ?? MIN_DRAWER_SIZE
```

Comment:
I realized that this calculation works incorrectly, as it returns min size only if `defaultSize` is not defined, but it should compare them and return min of them. Fixed, see below.